worker/cmd/main.go

```diff
@@ -31,40 +31,42 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net"
-	"os"
-	"strconv"
 	"strings"
 	"time"
 
 	"go.temporal.io/api/serviceerror"
 	"go.temporal.io/api/workflowservice/v1"
 	"go.temporal.io/sdk/activity"
 	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/log"
 	"go.temporal.io/sdk/worker"
 	"go.temporal.io/sdk/workflow"
 	"go.uber.org/zap"
 
 	"github.com/temporalio/maru/bench"
+	"github.com/temporalio/maru/common"
 	"github.com/temporalio/maru/target/basic"
 )
 
 func main() {
-	logger, err := zap.NewDevelopment()
+	zapLogger, err := zap.NewDevelopment()
 	if err != nil {
 		panic(err)
 	}
 
+	logger := NewZapAdapter(zapLogger)
+
 	logger.Info("Zap logger created")
-	namespace := getEnvOrDefaultString(logger, "NAMESPACE", client.DefaultNamespace)
-	hostPort := getEnvOrDefaultString(logger, "FRONTEND_ADDRESS", client.DefaultHostPort)
-	skipNamespaceCreation := getEnvOrDefaultBool(logger, "SKIP_NAMESPACE_CREATION", false)
+	namespace := common.GetEnvOrDefaultString(logger, "NAMESPACE", client.DefaultNamespace)
+	hostPort := common.GetEnvOrDefaultString(logger, "FRONTEND_ADDRESS", client.DefaultHostPort)
+	skipNamespaceCreation := common.GetEnvOrDefaultBool(logger, "SKIP_NAMESPACE_CREATION", false)
 
 	tlsConfig, err := getTLSConfig(hostPort, logger)
 	if err != nil {
-		logger.Fatal("failed to build tls config", zap.Error(err))
+		logger.Error("failed to build tls config", zap.Error(err))
```

Comment:
why changing it from Fatal to Error?

Reply:
Fatal is available in this context so I've put it back here. go.temporal.io/sdk/log Logger interface does not have Fatal